iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/FourSubstationsNodeBreakerFactory.java

```diff
@@ -0,0 +1,623 @@
+                .beginStep().setR(9.542847).setX(9.542842).setG(0.0).setB(0.0).setRho(1.0).setAlpha(32.26).endStep()
+                .beginStep().setR(16.263271).setX(16.263268).setG(0.0).setB(0.0).setRho(1.0).setAlpha(34.9).endStep()
+                .beginStep().setR(23.655737).setX(23.655735).setG(0.0).setB(0.0).setRho(1.0).setAlpha(37.54).endStep()
+                .beginStep().setR(31.720245).setX(31.720242).setG(0.0).setB(0.0).setRho(1.0).setAlpha(40.18).endStep()
+                .beginStep().setR(39.78473).setX(39.784725).setG(0.0).setB(0.0).setRho(1.0).setAlpha(42.8).endStep()
+                .add();
+
+        VscConverterStation vsc1 = s1vl2.newVscConverterStation()
+                .setId("VSC1")
+                .setName("VSC1")
+                .setNode(5)
+                .setLossFactor(1.1f)
+                .setReactivePowerSetpoint(500)
+                .setVoltageSetpoint(227.0)
+                .setVoltageRegulatorOn(false)
+                .add();
+        vsc1.getTerminal()
+                .setP(100.0)
+                .setQ(0.0);
+        vsc1.newReactiveCapabilityCurve()
+                .beginPoint()
+                .setP(5.0)
+                .setMinQ(-200.0)
+                .setMaxQ(100.0)
+                .endPoint()
+                .beginPoint()
+                .setP(100.0)
+                .setMinQ(-200.0)
+                .setMaxQ(100.0)
+                .endPoint()
+                .add();
+
+        LccConverterStation lcc1 = s1vl2.newLccConverterStation()
+                .setId("LCC1")
+                .setName("LCC1")
+                .setNode(25)
+                .setLossFactor(1.1f)
+                .setPowerFactor(0.6f)
+                .add();
+        lcc1.getTerminal()
+                .setP(100.0)
+                .setQ(50.0);
+
+        Substation s2 = network.newSubstation()
+                .setId("S2")
+                .setCountry(Country.ES)
+                .add();
+        VoltageLevel s2vl1 = s2.newVoltageLevel()
+                .setId("S2VL1")
+                .setNominalV(225.0)
+                .setLowVoltageLimit(0.0)
+                .setTopologyKind(TopologyKind.NODE_BREAKER)
+                .add();
+        BusbarSection busbarSectionS2VL1 = s2vl1.getNodeBreakerView().newBusbarSection()
+                .setId("S2VL1_BBS")
+                .setName("S2VL1_BBS")
+                .setNode(0)
+                .add();
+        createSwitch(s2vl1, "S2VL1_A", "S2VL1_A", SwitchKind.DISCONNECTOR, false, false, true, 0, 1);
+        createSwitch(s2vl1, "S2VL1_B", "S2VL1_B", SwitchKind.DISCONNECTOR, false, false, true, 0, 3);
+        createSwitch(s2vl1, "S2VL1_C", "S2VL1_C", SwitchKind.BREAKER, true, false, true, 1, 2);
+        createSwitch(s2vl1, "S2VL1_D", "S2VL1_D", SwitchKind.BREAKER, true, false, true, 3, 4);
+        createSwitch(s2vl1, "S2VL1_E", "S2VL1_E", SwitchKind.DISCONNECTOR, true, false, true, 0, 5);
```

Comment:
You should add a breaker for your line
On the schema connect it on the left of the S3VL1 to make it more readable